tooling/nargo_cli/build.rs

```diff
@@ -60,10 +60,20 @@ const IGNORED_BRILLIG_TESTS: [&str; 10] = [
     "is_unconstrained",
 ];
 
-/// Tests which aren't expected to work with the default inliner cases.
-const INLINER_MIN_OVERRIDES: [(&str, i64); 1] = [
+/// Tests which aren't expected to work with the default minimum inliner cases.
+const INLINER_MIN_OVERRIDES: [(&str, i64); 4] = [
     // 0 works if PoseidonHasher::write is tagged as `inline_always`, otherwise 22.
     ("eddsa", 0),
+    ("reference_counts_inliner_0", 0),
+    ("reference_counts_inliner_min", i64::MIN),
+    ("reference_counts_inliner_max", i64::MAX),
+];
+
+/// Tests which aren't expected to work with the default maximum inliner cases.
+const INLINER_MAX_OVERRIDES: [(&str, i64); 3] = [
+    ("reference_counts_inliner_0", 0),
+    ("reference_counts_inliner_min", i64::MIN),
+    ("reference_counts_inliner_max", i64::MAX),
```

Comment:
```suggestion
```
Why do we need this here? If we don't find a test `max_inliner` will just be set to `i64::MAX`, same with the min case above?

Reply:
I just thought it was clearer to specify all of these tests since they're only meant to be run on that one setting. I can remove them though if you'd prefer

Reply:
Let's just comment that, as I was confused to see `reference_counts_inliner_max` here.

Reply:
I thought that would be too messy for such a small thing so I went to remove the max/min cases from inliner max/min excludes instead but then figured I may as well make a separate list entirely for tests which should only be run on exactly 1 inliner setting.

Reply:
Yeah I like this more